app/models/comment.rb

```diff
@@ -379,62 +379,61 @@ def self.receive_tweet_without_using_since
 
   def self.check_and_add_tweets(tweets)
     tweets.each do |tweet|
-      if tweet.reply?
-        in_reply_to_tweet_id = tweet.in_reply_to_tweet_id
-        if in_reply_to_tweet_id.class == Fixnum
-          parent_tweet = Client.status(in_reply_to_tweet_id, tweet_mode: "extended")
-          parent_tweet_full_text = parent_tweet.attrs[:text] || parent_tweet.attrs[:full_text]
-          urls = URI.extract(parent_tweet_full_text)
-          node = get_node_from_urls_present(urls)
-          unless node.nil?
-            twitter_user_name = tweet.user.screen_name
-            tweet_email = find_email(twitter_user_name)
-            users = User.where(email: tweet_email)
-            if users.any?
-              user = users.first
-              replied_tweet_text = tweet.text
-              if tweet.truncated?
-                replied_tweet = Client.status(tweet.id, tweet_mode: "extended")
-                replied_tweet_text = replied_tweet.attrs[:text] || replied_tweet.attrs[:full_text]
-              end
-              replied_tweet_text = replied_tweet_text.gsub(/@(\S+)/){|m| "[#{m}](https://twitter.com/#{m})"}
-              replied_tweet_text = replied_tweet_text.gsub('@','')
-              comment = node.add_comment(uid: user.uid, body: replied_tweet_text, comment_via: 2, tweet_id: tweet.id)
-              comment.notify user
-            end
-          end
-        end
+      next unless tweet.reply?
+
+      in_reply_to_tweet_id = tweet.in_reply_to_tweet_id
+      next unless in_reply_to_tweet_id.class == Integer
```

Comment:
Oh did you mean these to be part of this?

Reply:
Yes, `Fixnum` is deprecated.

Reply:
no i meant the whole file refactor - it's fine, i just thought maybe you'd accidentally added in something from another PR!